assets/src/edit-story/utils/context.js

```diff
@@ -45,20 +45,20 @@ export const useContextSelector = (
 ) => {
   const ref = useRef();
 
+  const equalityFnCallback = useCallback(
+    (state) => {
+      const selected = selector(state);
+      if (equalityFn(ref.current, selected)) {
+        return ref.current;
+      }
+      ref.current = selected;
+      return selected;
+    },
+    [selector, equalityFn]
```

Comment:
There might be an issue here, but not certain. /cc @cvolzke4

selector (and possible equalityFn) change on every call. And thus we will be regenerating equalifyFnCallback each time. However, by nature, these callbacks are meant to be always reenterable. IMHO, we can and should always take the first version of these callbacks and use them for all calls. E.g. this useCallback should have deps of [].

Reply:
I'll file a bug for this.

Reply:
https://github.com/google/web-stories-wp/issues/1938

Reply:
Rather than "fixing" the deps to [], we can create refs to hold the always-current values of both `selector` and `equalityFn`. That's a nicer solution IMHO.

Reply:
Yeah, I think `refs` are indeed better. Let's go with that. @barklund could you pls make a note on the #1938 as well?